board/stm32h7/llfdcan.h

```diff
@@ -130,6 +130,14 @@ bool llcan_set_speed(FDCAN_GlobalTypeDef *CANx, uint32_t speed, uint32_t data_sp
 
     CANx->DBTP = (((sjw & 0xFU)-1U)<<FDCAN_DBTP_DSJW_Pos) | (((seg1 & 0x1FU)-1U)<<FDCAN_DBTP_DTSEG1_Pos) | (((seg2 & 0xFU)-1U)<<FDCAN_DBTP_DTSEG2_Pos) | (((prescaler & 0x1FU)-1U)<<FDCAN_DBTP_DBRP_Pos);
 
+    if (non_iso) {
+      // FD non-ISO mode
+      CANx->CCCR |= FDCAN_CCCR_NISO;
+    } else {
+      // FD ISO mode
+      CANx->CCCR &= ~(FDCAN_CCCR_NISO);
```

Comment:
It's better to move FD ISO reset to beginning of llcan_set_speed() (where we "reset operation mode to Normal" to be consistent with the code.